test/base/EtherCollateralLoansDepositCollateralTest.js

```diff
@@ -27,15 +26,13 @@ contract('EtherCollateralLoansDepositCollateralTest', function (accounts) {
         oracleInstance = await Mock.new();
         loanTermsConsInstance = await Mock.new();
         settingsInstance = await Mock.new();
-        marketsInstance = await Mock.new();
         atmSettingsInstance = await Mock.new();
         instance = await Loans.new();
         await instance.initialize(
             oracleInstance.address,
             lendingPoolInstance.address,
             loanTermsConsInstance.address,
             settingsInstance.address,
-            marketsInstance.address,
             atmSettingsInstance.address,
```

Comment:
I think we removed the atmSettings from Loans.initialize(...) due to it is in the settings contract.

Reply:
EtherCollateralLoans still has 5 params (1 that is empty). Needs to be changed then